src/DisplayOrientation.hpp

```diff
@@ -34,25 +34,18 @@ enum class DisplayOrientation : uint8_t {
   REVERSE_LANDSCAPE,
 };
 
-#ifdef KOBO
-/* Kobo defaults to portrait */
-static constexpr DisplayOrientation DEFAULT_DISPLAY_ORIENTATION =
-  DisplayOrientation::PORTRAIT;
-#else
-/* everything else defaults to landscape */
-static constexpr DisplayOrientation DEFAULT_DISPLAY_ORIENTATION =
+static DisplayOrientation default_display_orientation =
```

Comment:
I don't understand this whole commit, and the commit message doesn't explain anything.